src/views/virtualmachines/details/tabs/configuration/scripts/scripts-tab.scss

```diff
@@ -4,4 +4,17 @@
   &__buttons {
     margin-top: var(--pf-global--spacer--lg);
   }
+
+  .vm-description-item__title {
+    .pf-c-description-list__term {
+      font-family: var(--pf-global--FontFamily--heading--sans-serif);
+      font-size: var(--pf-global--FontSize--xl);
+      font-weight: var(--pf-global--FontWeight--normal);
+    }
+  }
+
+  .pf-c-divider {
+    margin-left: -1.5rem;
+    width: 102.5%;
+  }
```

Comment:
Yes. All dividers of amount 1 that are children of  components with `vm-scripts-tab` class.

Reply:
I don't think we should have that.
In my opinion the current dividers are great and with this style they seems weird

Reply:
We need it to achieve the expected look according to the design doc I've received.

Reply:
I've reviewed the doc: https://docs.google.com/document/d/1VOjS0M-8YF_pefXaLWcqFYtoJGWFAQdSOJ3CsxnpY7M/edit#heading=h.ozvp0t9knadz

I'm okay with the fact that the divider should take all the width as in the environment tab. I've added a max-width to that page just to not have the title and texts shifting between editor on/off.
But what you are trying to do here is to stretch the divider on the left and I don't think it's what Yifat requested.

Reply:
Yes, she did.

Reply:
Yes, I've seen that and I'm saying that for `dividers length` I'm pretty sure she means the width on the right. Currently, the content of that page is limited approximately by `700px` and the divider does not take the whole page width for this reason.

As we can see from her design of the script tab, the divider is not shifted to the left with respect to the current implementation and takes all the width until the end without any limitations. 

Disks and Diagnostics tabs have tables and those lines are to limit rows. They are not Dividers.

Reply:
> Yes, I've seen that and I'm saying that for dividers length I'm pretty sure she means the width on the right. 

The width is usually is related to the content of the page. If that takes just a small part of the whole page (as in our case), the divider simply is shorter. You can see that also in other pages.

Maybe it could have been a bit longer than I've set (just a bit more than 100%, to keep the original length on the right because of moving it to the left, however the exact expected length wasn't explicitly specified), but the main issue there was that the divider wasn't long enough in the left. PTAL at Disks or Diagnostics tabs again to see where the dividers start.

I can eventually ask Yifat about the exact length of the divider and to force it to be even longer despite its default behavior, but we definitely need to start it from the left - same as in other pages with more subsections I've mentioned earlier, for consistency reasons.

> As we can see from her design of the script tab, the divider is not shifted to the left with respect to the current implementation and takes all the width until the end without any limitations.

Thanks for mentioning it. Unfortunately, not everything is always properly updated and displayed in the design docs, as these things are changing often. This is why I talk to Yifat on a daily basis so I know what to do.

> Disks and Diagnostics tabs have tables and those lines are to limit rows. They are not Dividers.

From the visual point of view, it's the same component.

> PS: Diagnostics does not have any padding like what we have on other pages.

Not sure what exactly you mean by this but that page has more issues that should be fixed (followup PR), according to how the components are displayed in that page.